packages/scrum-server/src/api/components/auth/controller.js

```diff
@@ -55,12 +55,16 @@ const authUserFacebookRedirect = async (req, res) => {
 };
 
 const authLoginSucess = async (req, res) => {
-  if (req.user) {
+  const { user } = req;
+  const copy = user.toJSON();
+  // eslint-disable-next-line no-underscore-dangle
+  copy.id = copy._id;
+  if (user) {
```

Comment:
is that if necessary  ? 
2 lines above I can see this:

```  
const copy = user.toJSON();
  // eslint-disable-next-line no-underscore-dangle
  copy.id = copy._id;
  ```
  
  So or the user is always present or that will fail in some scenarios